components/sup/src/manager.rs

```diff
@@ -1370,14 +1377,16 @@ impl Manager {
     /// # Locking (see locking.md)
     /// * `MemberList::entries` (read)
     /// * `RumorStore::list` (write)
-    fn restart_elections_rsw_mlr(&mut self, feature_flags: FeatureFlag) {
-        self.butterfly.restart_elections_rsw_mlr(feature_flags);
+    /// * `ManagerServices::inner` (read)
+    /// * `ManagerServices::inner` (write)
+    fn restart_elections_rsw_mlr_msr(&mut self, feature_flags: FeatureFlag) {
+        self.butterfly.restart_elections_rsw_mlr_msr(feature_flags);
     }
 
     /// # Locking (see locking.md)
     /// * `GatewayState::inner` (write)
-    fn stop_service_gsw(&mut self, ident: &PackageIdent, shutdown_input: &ShutdownInput) {
-        if let Some(service) = self.remove_service_from_state(&ident) {
+    fn stop_service_gsw_msw(&mut self, ident: &PackageIdent, shutdown_input: &ShutdownInput) {
```

Comment:
Missing a write annotation here, maybe the one from above?

Reply:
Perhaps! Thanks for noticing that